### DISCUSSION TITLE
1.5.3

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#1045] [New Release]
### Changelog
- Optimize logcat
- Fix manager crash when scrolling to the bottom of an empty log
- Fix crash of randomized Magisk manager
### 更新日志
- 优化日志
- 修复管理器在滚动到空日志的底部时候崩溃
- 修复随机包名的 Magisk 崩溃

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.5.3'>1.5.3</a>.</em>

**Comment by 2833291831** [comment]
永远的神
